sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/ApiExtensions.kt

```diff
@@ -252,3 +253,9 @@ private fun KtParameter.determineType(bindingContext: BindingContext) =
 
 fun KtParameter.determineTypeAsString(bindingContext: BindingContext, printTypeArguments: Boolean = false) =
     determineType(bindingContext)?.getJetTypeFqName(printTypeArguments)
+
+fun KtNamedFunction.returnTypeAsString(bindingContext: BindingContext, printTypeArguments: Boolean = false) =
+    returnType(bindingContext)?.getJetTypeFqName(printTypeArguments)
+
+fun KtNamedFunction.returnType(bindingContext: BindingContext) =
```

Comment:
It looks like this function could be private. Also. maybe it makes sense to combine this function with `returnTypeAsString`?

Reply:
My idea behind this was that you may want to perform some further processing on the type, for which you might need the actual `KotlinType`. So by splitting it up like this, you still have that option in the API. WDYT?